Лабораторная работа1/task1.4.1.py

```diff
@@ -0,0 +1,11 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+sum_before=sum(numbers[:4])
+sum_after=sum(numbers[5:])
+len_before=len(numbers[:4])
+len_after=len(numbers[4:])
```

Comment:
В данном случае можно было найти длину всего списка, так как там по заданию, среднее арифметическое вычисляется от длины исходного списка
```suggestion
len_list=len(numbers)
```